modules/sage/1.0/config/default.yaml

```diff
@@ -0,0 +1,43 @@
+lcr-modules:
+    
+    sage:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            
+
+        scratch_subdirectories: []
+
+        # include here any additional flags to modify default parameters
+        options:
+            sage_run: ""
+
+
+        conda_envs:
+            sage: "{MODSDIR}/envs/hmftools-sage-2.2.yaml"
+            bcftools: "{MODSDIR}/envs/bcftools-1.10.2.yaml"
+            
+        resources:
+            sage_run:
+                mem_mb: 60000
+                disk_mb: 20000 # Will ensure at least this much space is left
```

Comment:
reserving/expecting 20G per sample seems a bit high, no?

Reply:
the largest file from the output is 3.5 Gb, but because at some point both compressed and decompressed files may exist (right before the _sage_run is completed), I gave it more space to accommodate both of these files. But I might decrease that further down if needed.